sn_node_manager/src/bin/cli/main.rs

```diff
@@ -284,11 +284,22 @@ pub enum SubCmd {
     /// sudo if you defined system-wide services; otherwise, do not run the command elevated.
     #[clap(name = "start")]
     Start {
+        /// The max time in milliseconds to wait for the RPC connection to the node to be established.
+        ///
+        /// Setting this will automatically scale the interval between each service launch. This argument
```

Comment:
I was thinking about this a little bit.

"Setting this will automatically scale the interval between each service launch."

To me, this language implies that the dynamic scaling is only going to apply if you used this argument. However, that's not the case, because there is a default value for the connection timeout. So, the dynamic scaling is really going to apply unless it gets overridden by the `--interval` argument.

I think it could possibly make more sense for the dynamic scaling to be explain on the start command itself, rather than on this connection timeout argument. I'm not really sure there should actually be *any* link between this connection timeout and the dynamic scaling.